src/EnergyPlus/DataLoopNode.hh

```diff
@@ -262,12 +309,26 @@ namespace DataLoopNode {
 			TempSetPointHi( TempSetPointHi ),
 			TempSetPointLo( TempSetPointLo ),
 			Height( Height ),
+			SchedOutAirDryBulb ( SchedOutAirDryBulb ),
```

Comment:
@XuanLuoLBL In most of the code, the member constructors have been deleted (they were initially added in the original translation to c++).  I see there are a few places that still have them, like here.  @Myoldmopar Is there any reason to keep these?  If not, we should make a separate PR to get rid of them all.